Templates/ManualTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+    "defaultFunctionName": "ManualTriggerCSharp",
+    "description": "A C# function without triggers, inputs, or outputs",
```

Comment:
Let's update the description of this and the Node one to be something like:

"A C# function that is triggered manually via the portal"

Since it's confusing to say "without triggers" when it is called "Manual Trigger"